src/ctl/src/cmd_impl/trace.rs

```diff
@@ -41,6 +41,7 @@ pub async fn trace(context: &CtlContext) -> anyhow::Result<()> {
         let StackTraceResponse {
             actor_traces,
             rpc_traces,
+            ..
```

Comment:
> > Will this PR also enable shared buffer compaction traces in CN
> 
> No. Shall we?

+1. But since the performance overhead is unkonwn and shared buffer compaction may have bigger impact on streaming performance, we can leave it to a separate PR and verify whether the overhead is acceptable like #10144.